### PR TITLE
talk about special features in use-case

### PR DESCRIPTION
**Pull request by AurelienJaquier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +139,8 @@ To get a list with all the available feature names
 efel.get_feature_names()
 ```
 
+Note that the extra-cellular features, the bpap_attenuation feature and the check_ais_initiation feature are not listed above because they have to be used in a special way, as described [here](https://efel.readthedocs.io/en/latest/extrafeats_example.html) for extra-cellular features, [here](https://efel.readthedocs.io/en/latest/eFeatures.html#bpap-attenuation) for bpap_attenuation feature and [here](https://efel.readthedocs.io/en/latest/eFeatures.html#check-ais-initiation) for check_ais_initiation feature.
```
Comment (darshanmandge) [inline README.md:142]: Can we give links to Github [source](https://github.com/BlueBrain/eFEL/blob/master/docs/source/eFeatures.rst#efeature-descriptions) if the readthedocs do not work for some reason?

File: docs/source/python_example1.rst [hunk 2]
```diff
@@ -9,6 +9,9 @@ To get a list with all the available eFeature names::
 
     efel.get_feature_names()
 
+Note that the extra-cellular features, the bpap_attenuation feature and the check_ais_initiation feature are not listed above because they have to be used in a special way,
+as described `here <https://efel.readthedocs.io/en/latest/extrafeats_example.html>`_ for extra-cellular features, `here <https://efel.readthedocs.io/en/latest/eFeatures.html#bpap-attenuation>`_ for bpap_attenuation feature and `here <https://efel.readthedocs.io/en/latest/eFeatures.html#check-ais-initiation>`_ for check_ais_initiation feature.
```
Comment (darshanmandge) [inline docs/source/python_example1.rst:13]: same here

File: README.md (hunk 1 above)
Comment (AurelienJaquier) [reply README.md]: done in latest commit

**Review by darshanmandge — APPROVED** [review]
:+1:

**Pull request merged**